app/models/changelog_data.php

```diff
@@ -59,9 +59,97 @@
     '4.4'   => '2016-06-02',
     '5.0'   => '2016-08-04',
     '5.1'   => '2016-11-04',
+    '5.2'   => '2017-02-02',
 ];
 
 $changelog = [
+  '5.2' => [
+      [
+          'message' => ['Added link to Pootle or Pontoon under strings from the Aurora channel, in order to quickly edit translations'],
```

Comment:
Maybe: "For locales working on Pootle or Pontoon, added links under Aurora search results to quickly access translations in the tool."